cmd/system-probe/main_windows.go

```diff
@@ -13,7 +13,7 @@ func main() {
 	flag.BoolVar(&opts.version, "version", false, "Print the version and exit")
 	flag.Parse()
 
-	runAgent()
+	runAgent(make(chan struct{}))
```

Comment:
Is the windows version not handling signals?

Reply:
Hmm, that's a good point. I left it out because my understanding of @leeavital's comment above was that we didn't want a case where we had two things handling a termination of the probe. 

Part of this is my ignorance of the windows service manager, which is how the system probe on windows is going to be handled for startup/shutdown/etc. I am unsure if the service manager will handle signals to the probe code. I'll look into this and get back to you.  

Just for reference, the system-probe will be controlled similarly to the process-agent: https://github.com/DataDog/datadog-agent/blob/master/cmd/process-agent/main_windows.go

Reply:
Discussed over DM - it will be handle depending on whether or not this is an interactive session. 

Will add the os handlers after the session management code has been added by @derekwbrown.